### PR TITLE
159719959 ch2 add delete functianlity

### PR DESCRIPTION
**Pull request by TheSteelGuy**
**Introduction**
This PR develops delete question endpoint

**Description of Task to be completed:**
On sending Delete requests. appropriate responses are to be received

**How to manually test**
- $ pip install virtualenv 
- NB This install virtualenv Globally(virtual environment allows you to create independent envs isolated from  your actual physical machine)
- $ virtualenv -p python3 myenv 
- $ source myenv/bin/activate
- Clone this repository: _$ git clone https://github.com/TheSteelGuy/StackOverFlowLiteApi.git_
- navigate into the repository after cloning
- $ git checkout 159719959-ch2-add-delete-functianlity to switch from master to this branch
- $ pip install -r requirements.txt
-  Launch postman
* #### Fetch all questions.
    `DELETE /api/v1/questions/questionId`
    ```
    headers = {content_type:application/json}
    ```

* #### Fetch a specific question.   
    `GET /api/v1/questions/<questionId>` 
    ```
    headers = {content_type:application/json} 
    ```

**Any background information**

-  Successfull request returns JSON formatted response appropriately

**Relevant story**
[Click here](https://www.pivotaltracker.com/n/projects/2189398/stories/159719959)

### CODE REVIEW

**Comment by Tony-Ndichu** [comment]
Hey, good job there, side note: I tested getting all questions and it came up empty, maybe add some dummy data just  to show that it actually works?

**Comment by p8ul** [comment]
Great stuff

**Comment by TheSteelGuy** [comment]
How empty, coz when you fetch questions and there are none you will be alerted that no question exist

File: api/question/views.py [hunk 1]
```diff
@@ -37,14 +37,32 @@ def post(self):
 
     def get(self):
```
Comment (TheSteelGuy) [inline api/question/views.py]: @Tony-Ndichu  the problem is here, thanks for noting that, i think its fixed on the other prs

**Pull request merged**